app/mailers/notifications.rb

```diff
@@ -67,4 +73,8 @@ def no_reply_email_address
     address.display_name = name
     address.format
   end
+
+  def content_second_line_email_address
+    "second-line-content@digital.cabinet-office.gov.uk"
```

Comment:
No reason, maybe we should do but now it's in a public PR! I guess it's also not really a sensitive email address whereas the Watchkeeper one sounds like it may be.